packages-native/popup-menu/src/__tests__/PopupMenu.spec.tsx

```diff
@@ -0,0 +1,97 @@
+import { PopupMenuProps } from "../../typings/PopupMenuProps";
+import { PopupMenuStyle } from "../ui/Styles";
+import { Text, TouchableHighlight } from "react-native";
+import { createElement } from "react";
+import { actionValue } from "@native-mobile-resources/util-widgets";
+import { fireEvent, render } from "react-native-testing-library";
+import { PopupMenu } from "../PopupMenu";
+import { MenuItem } from "react-native-material-menu";
+
+let dummyActionValue: any;
+let defaultProps: PopupMenuProps<PopupMenuStyle>;
+
+describe("Popup menu", () => {
+    beforeEach(() => {
+        dummyActionValue = actionValue();
+        defaultProps = {
+            renderMode: "basic",
+            name: "popup-menu",
+            style: [],
+            menuTriggerer: undefined,
+            basicItems: [
+                { itemType: "item", action: dummyActionValue, caption: "yolo", styleClass: "default" },
+                { itemType: "divider", styleClass: "default" }
+            ],
+            complexItems: [{ content: <Text>Yolo</Text>, action: dummyActionValue }]
+        };
+    });
+    it("renders menu triggerer", () => {
+        const menuTriggerer = <Text>Menu Triggerer</Text>;
+        const component = render(<PopupMenu {...defaultProps} menuTriggerer={menuTriggerer} />);
+
+        expect(component.toJSON()).toMatchSnapshot();
+    });
+
+    describe("with basic items", () => {
+        it("renders", () => {
+            const component = render(<PopupMenu {...defaultProps} />);
+
+            expect(component.toJSON()).toMatchSnapshot();
```

Comment:
I see that web tests (e.g.in  #173) don't use `toJSON` and still have ok snapshots. Is it web vs native difference?

Reply:
Yes indeed, though my previous comment still persists. We are using react-native-test-renderer in native